src/data/repositories/LandingNodeDefaultRepository.ts

```diff
@@ -32,7 +32,12 @@ export class LandingNodeDefaultRepository implements LandingNodeRepository {
                 : [];
 
             const validations = roots.map(root =>
-                LandingNodeModel.decode(buildDomainLandingNode(root, _.flatten(persisted)))
+                LandingNodeModel.decode(
+                    buildDomainLandingNode(
+                        root,
+                        persisted.find(model => model.find(item => item.parent === root.id)) ?? []
```

Comment:
I see the point of passing only the items on the same array as the root is 👌🏼. That way, only sections owned by the root are built on the root. But it seems the key issue is that, for some reason, there is an identic section repeated in each one of the root's `[]` in the `dataStore/landing-pages`; that's why doing `_.flatten` was making 5 repeated sections when landing on home page app. Another patch would be `_(persisted).flatten().uniqBy(({ id }) => id).value()`, because the problem is the duplicates.

So here there are two key points on the real problem: we shouldn't have duplicates in the `dataStore`, and we shouldn't have a section/sub-section/category in another's root array that doesn't belong to it.

@ifoche suggested that this could be happening due to imports/cloning landing pages. Indeed, I did try in another env "create landing -> create section -> export landing -> import landing"; and it was populating the created section to each landing on the `dataStore` with the same `id`. I don't know what approach would be the best. Maybe @xurxodev can give a hint about "where the validation should be happening"; if it should happen not only when importing landings, but also when retrieving the nodes (maybe on the settings page) that there is an inconsistency on the `dataStore` that should be fixed.